toolbox/deps_update/util.go

```diff
@@ -0,0 +1,44 @@
+// Copyright 2017 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"crypto/md5"
+	"encoding/hex"
+	"fmt"
+	"log"
+	"os/exec"
+	"strings"
+)
+
+// GetMD5Hash generates an MD5 digest of the given string
+func GetMD5Hash(text string) string {
+	hash := md5.Sum([]byte(text))
+	return hex.EncodeToString(hash[:])
```

Comment:
The [:] operator creates a slice (type []byte) from an array (type [16]byte). (See full explanation [here](https://blog.golang.org/go-slices-usage-and-internals)) It is necessary to make such conversion here since md5.Sum() returns [16]byte but hex.EncodeToString wants []byte.

I think this function is generically useful to not just this particular feature/script but also the rest of test-infra so putting it under util is in my opinion a good idea.